www-apps/bugzilla/bugzilla-5.0.6.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit webapp eutils # should not depend on a web server
+
+DESCRIPTION="Bugzilla is the Bug-Tracking System from the Mozilla project"
+SRC_URI="https://ftp.mozilla.org/pub/mozilla.org/webtools/${P}.tar.gz"
+HOMEPAGE="https://www.bugzilla.org"
+
+LICENSE="MPL-2.0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="mysql postgres sqlite test" #modperl graphviz are not suitable for USE as per https://devmanual.gentoo.org/general-concepts/use-flags/
+RESTRICT="!test? ( test )" #why restrict tests ?
```

Comment:
Yeah, definitely put a comment in if we need to keep them restricted, but from IRC, sounds like you've got them running now?